src/apps/users/services/core.py

```diff
@@ -66,8 +74,18 @@ async def send_password_recovery(self, schema: PasswordRecoveryRequest) -> Publi
 
         exp = settings.authentication.password_recover.expiration // 60
 
+        # Default to web frontend
+        frontend_base_url = settings.service.urls.frontend.web_base
+        if content_source == MindloggerContentSource.admin:
+            # Change to admin frontend if the request came from there
+            frontend_base_url = settings.service.urls.frontend.admin_base
+
+        base_url_with_protocol = frontend_base_url
+        if not base_url_with_protocol.startswith("http"):
+            base_url_with_protocol = f"https://{base_url_with_protocol}"
```

Comment:
interpolate with protocol only in line 88 as original code using frontend_base_domain var

Reply:
Same comment as [here](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/pull/1100#discussion_r1491718601)